packages/e2e-testing/src/payment-server.ts

```diff
@@ -304,14 +304,21 @@ function startApp(
           indexer: joi
             .object({
               url: joi.string().required(),
-              id: joi.string().required()
```

Comment:
Can you explain this change? Is it the case that the contracts package has changed, which necessitates a new schema for allocations?

If yes, I think that information is important to include in this git history.

Reply:
Rebased with clearer commit history.